pkg/asset/tls/mcscertkey.go

```diff
@@ -37,7 +41,18 @@ func (a *MCSCertKey) Generate(dependencies asset.Parents) error {
 		Subject:      pkix.Name{CommonName: hostname},
 		ExtKeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		Validity:     ValidityTenYears,
-		DNSNames:     []string{hostname},
+	}
+
+	switch installConfig.Config.Platform.Name() {
+	case openstacktypes.Name:
+		apiIntIP, err := cidr.Host(&installConfig.Config.Networking.MachineCIDR.IPNet, 5)
```

Comment:
this should not be calucated when these fields exist in InsallConfig.
https://github.com/openshift/installer/pull/1998/commits/f5f4125f5a51a4b15680196e6c22e671ae793bde#diff-26eeddc74b12e6afb9e2932066a9b49fR33

Reply:
fixed